### ISSUE TITLE
refactor OWL sublogic and connect sublogic specific provers


### ISSUE THREAD

**Issue by sternk** [spechub/Hets#931]
The currently defined sublogics are given in <a href="https://github.com/spechub/Hets/blob/master/OWL/Sublogic.hs">spechub/Hets:OWL/Sublogic.hs</a> and the provers are fact++ and pellet
For EL, connect at least one of the provers described in http://www.semantic-web-journal.net/content/new-submission-comparison-reasoners-large-ontologies-owl-2-el-profile

There should be a sublogic "OWL with restrictions" (like no asymmytry and transitivity for a role simultaneously), and the provers should be attached to that sublogic (or a suitable sublogic of it).

**Comment by tillmo** [comment]
#1865 is related

**Comment by mcodescu** [comment]
#1863 is related

**Comment by mcodescu** [comment]
Also add SWRL, see #1847

**Comment by b-gehrke** [comment]
Check Fact++
Add Racer
Add [ELK](https://www.uni-ulm.de/en/elkreasoner/)
Add Conclude

**Comment by tillmo** [comment]
@b-gehrke has the sublogic part of this issue been done?

**Comment by b-gehrke** [comment]
Yes. #2058 adds the `unrestrictedDL` and `rules` flag to `OWL2.Sublogic.OWLSub`
